viewport/viewport_test.go

```diff
@@ -0,0 +1,41 @@
+package viewport
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_countHeightBasedOnWidth(t *testing.T) {
+	t.Parallel()
+	tests := map[string]struct {
+		lines []string
+		width int
+		want  int
+	}{
+		"Empty lines": {
+			lines: []string{},
+			width: 0,
+			want:  0,
+		},
+		"Lines within width": {
+			lines: []string{"123", "123"},
+			width: 5,
+			want:  2,
+		},
+		"Lines over width": {
+			lines: []string{"1234567890", "123"},
+			width: 5,
+			want:  3,
+		},
+	}
+
+	for name, tt := range tests {
+		tt := tt
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
+			got := countHeightBasedOnWidth(tt.lines, tt.width)
+			assert.Equal(t, tt.want, got)
```

Comment:
```suggestion
			if tt.want != got {
				t.Errorf("expected %v, got %v", tt.want, got)
			}
```

so we can drop the testify department

Reply:
Thank you for reviewing!
I'm done in ee209540653a.